docs/_layouts/en.html

```diff
@@ -11,7 +11,7 @@
           <a href="{{ '/' | relative_url }}">Home</a>
         </li>
         <li class="nav-item dropdown-trigger">
-          <a href="{{ '/basics' | relative_url }}">Guides</a>
+          <a href="{{ '/guides' | relative_url }}">Guides</a>
```

Comment:
still not 100% sure this is a good idea, trying to provide a compact guide with an abstract project to give an simpler overview about Leoric features.

Reply:
I think an example project more useful than a good document☺️

Reply:
agree, will expand this documentation into an example project :-)